modules/markup/renderer.go

```diff
@@ -63,7 +73,8 @@ type RenderContext struct {
 	GitRepo          *git.Repository
 	ShaExistCache    map[string]bool
 	cancelFn         func()
-	TableOfContents  []Header
+	SidebarTocTree   ast.Node
```

Comment:
We could even think about dropping the `-Tree`:
`SidebarTOC`/`SidebarToc`

Reply:
Sure. 28f4b8dbdbe39817ba338769e2e5b316436909a8

I think sidebarTocNode / sidebarTocContent could be clearer than before, and `sidebarTocContent` matches the main `content` name